w_common/lib/src/common/cache/cache.dart

```diff
@@ -150,22 +150,22 @@ class Cache<TIdentifier, TValue> extends Object with Disposable {
 
   /// The stream of [CacheContext]s that dispatches when an item is updated in
   /// the cache.
-  Stream<CacheContext<TIdentifier, TValue>> get didUpdate =>
+  Stream<CacheContext<TIdentifier, TValue?>> get didUpdate =>
```

Comment:
The type of `_didUpdateController.stream` is `StreamController<CacheContext<TIdentifier, TValue>>`. I don't think we need `TValue` to be nullable here.

```suggestion
  Stream<CacheContext<TIdentifier, TValue>> get didUpdate =>
```